editor.planx.uk/src/testUtils.tsx

```diff
@@ -29,6 +19,15 @@ export const setup = (
   ...render(<ThemeProvider theme={defaultTheme}>{jsx}</ThemeProvider>),
 });
 
+// export const axe = configureAxe({
+//   rules: {
+//     // Currently, jest-axe does not correctly evaluate this rule due to an issue with jsdom
+//     // https://github.com/dequelabs/axe-core/issues/2587
+//     // To pass this test, non-decorative MUI icons should always use the 'titleAccess' prop
+//     "svg-img-alt": { enabled: false },
+//   },
+// });
```

Comment:
Nice to know we don't need this - if we see an error with `titleAccess` again or failing a11y tests we can revisit.

Reply:
Will properly delete in follow-up PRs rather than leave commented-out here !